packages/server/api/src/app/flows/flow/flow.service.ts

```diff
@@ -683,6 +670,21 @@ const getConnections = async (
   });
 };
 
+export async function assertThatFlowIsNotInternal(flow: Flow): Promise<void> {
+  if (flow.isInternal) {
+    const message =
+      'Flow is internal, cannot be manipulated through the flow API.';
+    logger.warn(message, {
```

Comment:
adding a logger for it because this should not happen, if we get here it means a user used the API with a flow id manually